ci/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+yarn --cwd ui install
+yarn --cwd ui build
+yarn --cwd ui test
+
+export GO111MODULE="on"
+go get -u github.com/rakyll/statik
+go generate github.com/aerokube/selenoid-ui
```

Comment:
you removed the ui package where generator is executed, so no need to keep it

Reply:
Needed to run statik: https://github.com/aerokube/selenoid-ui/blob/master/main.go#L25 Explicit call was removed.

Reply:
```suggestion
```

Reply:
@lanwen removing this line makes no sense.

Reply:
true